packages/foam-vscode/src/services/variable-resolver.ts

```diff
@@ -1,9 +1,11 @@
 import { findSelectionContent } from './editor';
 import { window } from 'vscode';
 import { UserCancelledOperation } from './errors';
+import slugger from 'github-slugger';
```

Comment:
rather than importing the slugger, can you go through `utils/slug.ts`?

Reply:
I made the changes, is that what you had in mind?